TypeScript.py

```diff
@@ -814,6 +840,7 @@ def on_selection_modified(self, view):
                 self.setOnSelectionIdleTimer(1250)
             else:
                 self.setOnSelectionIdleTimer(50)
+            self.showTopLineStatus(info, view)
```

Comment:
Agree.  I just removed it entirely from on_selection_modified because the pulse timer handles it anyway.